// prototyped.js@2.0.0/dist/utils.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.OPERATOR = exports.deepClone = exports.filter = exports.addPrototype = exports.pathToKeys = void 0;
const hasOwnProperty = Object.prototype.hasOwnProperty;
/**
 * convert key path string to key array
 * @private
 * @param {String} path
 * @returns {Array}
 * @example
 * pathToKeys("selector.to[0][11].value"); // ["selector", "to", 0, 11, "value"]
 */
const pathToKeys = (path) => path
    .replace(/([^.])\[/g, (m, match) => `${match}.[`)
    .split(".")
    .map((key) => {
    const match = key.match(/^\[(\d+)]$/);
    if (match)
        return +match[1];
    return key;
});
exports.pathToKeys = pathToKeys;
/**
 *
 * @private
 * @param {*} obj
 * @param {String} key
 * @param {*} method
 * @example
 * addPrototype(Object, "$size", function() {return this;});
 */
const addPrototype = (obj, key, method) => {
    const prototype = obj.prototype;
    if (hasOwnProperty.call(prototype, key))
        return;
    Object.defineProperty(prototype, key, {
        value(...args) {
            return method.apply(0, [this].concat(args));
        },
        writable: true,
    });
};
exports.addPrototype = addPrototype;
/**
 *
 * @private
 * @param {Array} arr
 * @param {String} [path]
 * @param {Function} fn
 * @returns {*}
 * @example
 * filter([1, 2, 3], undefined, (value) => value > 1);
 */
const filter = (arr, path, fn) => {
    if (path) {
        const keys = exports.pathToKeys(path);
        const reducer = (item) => keys.reduce((prev, curr) => prev[curr], item);
        return arr.filter((item, index, items) => fn(reducer(item), index, items));
    }
    return arr.filter(fn);
};
exports.filter = filter;
/**
 *
 * @private
 * @param {*} value
 * @returns {*}
 * @example
 * deepClone([1, 2, 3]);
 */
const deepClone = (value) => {
    if (typeof value !== "object")
        return value;
    if (value instanceof Date)
        return new Date(value.getTime());
    const clone = Object.assign({}, value);
    Object.keys(clone).forEach((key) => (clone[key] =
        typeof value[key] === "object"
            ? exports.deepClone(value[key])
            : value[key]));
    if (Array.isArray(value)) {
        clone.length = value.length;
        return Array.from(clone);
    }
    return clone;
};
exports.deepClone = deepClone;
var OPERATOR;
(function (OPERATOR) {
    OPERATOR["LT"] = "<";
    OPERATOR["LTE"] = "<=";
    OPERATOR["EQ"] = "=";
    OPERATOR["NE"] = "<>";
    OPERATOR["GTE"] = ">=";
    OPERATOR["GT"] = ">";
})(OPERATOR = exports.OPERATOR || (exports.OPERATOR = {}));
