// prototyped.js@2.0.0/dist/object/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.values = exports.snakeCaseKeys = exports.size = exports.set = exports.reduce = exports.omit = exports.merge = exports.mapValues = exports.mapKeys = exports.map = exports.lowerCaseKeys = exports.keys = exports.kebabCaseKeys = exports.isPlainObject = exports.isObjectLike = exports.isObject = exports.isEmpty = exports.invert = exports.get = exports.forEach = exports.flatten = exports.equals = exports.empty = exports.dig = exports.defaults = exports.clone = exports.camelCaseKeys = exports.assign = void 0;
var assign_1 = require("./assign");
Object.defineProperty(exports, "assign", { enumerable: true, get: function () { return __importDefault(assign_1).default; } });
var camelCaseKeys_1 = require("./camelCaseKeys");
Object.defineProperty(exports, "camelCaseKeys", { enumerable: true, get: function () { return __importDefault(camelCaseKeys_1).default; } });
var clone_1 = require("./clone");
Object.defineProperty(exports, "clone", { enumerable: true, get: function () { return __importDefault(clone_1).default; } });
var defaults_1 = require("./defaults");
Object.defineProperty(exports, "defaults", { enumerable: true, get: function () { return __importDefault(defaults_1).default; } });
var dig_1 = require("./dig");
Object.defineProperty(exports, "dig", { enumerable: true, get: function () { return __importDefault(dig_1).default; } });
var empty_1 = require("./empty");
Object.defineProperty(exports, "empty", { enumerable: true, get: function () { return __importDefault(empty_1).default; } });
var equals_1 = require("./equals");
Object.defineProperty(exports, "equals", { enumerable: true, get: function () { return __importDefault(equals_1).default; } });
var flatten_1 = require("./flatten");
Object.defineProperty(exports, "flatten", { enumerable: true, get: function () { return __importDefault(flatten_1).default; } });
var forEach_1 = require("./forEach");
Object.defineProperty(exports, "forEach", { enumerable: true, get: function () { return __importDefault(forEach_1).default; } });
var get_1 = require("./get");
Object.defineProperty(exports, "get", { enumerable: true, get: function () { return __importDefault(get_1).default; } });
var invert_1 = require("./invert");
Object.defineProperty(exports, "invert", { enumerable: true, get: function () { return __importDefault(invert_1).default; } });
var isEmpty_1 = require("./isEmpty");
Object.defineProperty(exports, "isEmpty", { enumerable: true, get: function () { return __importDefault(isEmpty_1).default; } });
var isObject_1 = require("./isObject");
Object.defineProperty(exports, "isObject", { enumerable: true, get: function () { return __importDefault(isObject_1).default; } });
var isObjectLike_1 = require("./isObjectLike");
Object.defineProperty(exports, "isObjectLike", { enumerable: true, get: function () { return __importDefault(isObjectLike_1).default; } });
var isPlainObject_1 = require("./isPlainObject");
Object.defineProperty(exports, "isPlainObject", { enumerable: true, get: function () { return __importDefault(isPlainObject_1).default; } });
var kebabCaseKeys_1 = require("./kebabCaseKeys");
Object.defineProperty(exports, "kebabCaseKeys", { enumerable: true, get: function () { return __importDefault(kebabCaseKeys_1).default; } });
var keys_1 = require("./keys");
Object.defineProperty(exports, "keys", { enumerable: true, get: function () { return __importDefault(keys_1).default; } });
var lowerCaseKeys_1 = require("./lowerCaseKeys");
Object.defineProperty(exports, "lowerCaseKeys", { enumerable: true, get: function () { return __importDefault(lowerCaseKeys_1).default; } });
var map_1 = require("./map");
Object.defineProperty(exports, "map", { enumerable: true, get: function () { return __importDefault(map_1).default; } });
var mapKeys_1 = require("./mapKeys");
Object.defineProperty(exports, "mapKeys", { enumerable: true, get: function () { return __importDefault(mapKeys_1).default; } });
var mapValues_1 = require("./mapValues");
Object.defineProperty(exports, "mapValues", { enumerable: true, get: function () { return __importDefault(mapValues_1).default; } });
var merge_1 = require("./merge");
Object.defineProperty(exports, "merge", { enumerable: true, get: function () { return __importDefault(merge_1).default; } });
var omit_1 = require("./omit");
Object.defineProperty(exports, "omit", { enumerable: true, get: function () { return __importDefault(omit_1).default; } });
var reduce_1 = require("./reduce");
Object.defineProperty(exports, "reduce", { enumerable: true, get: function () { return __importDefault(reduce_1).default; } });
var set_1 = require("./set");
Object.defineProperty(exports, "set", { enumerable: true, get: function () { return __importDefault(set_1).default; } });
var size_1 = require("./size");
Object.defineProperty(exports, "size", { enumerable: true, get: function () { return __importDefault(size_1).default; } });
var snakeCaseKeys_1 = require("./snakeCaseKeys");
Object.defineProperty(exports, "snakeCaseKeys", { enumerable: true, get: function () { return __importDefault(snakeCaseKeys_1).default; } });
var values_1 = require("./values");
Object.defineProperty(exports, "values", { enumerable: true, get: function () { return __importDefault(values_1).default; } });
