// prototyped.js@2.0.0/dist/math/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.radsToDegrees = exports.primes = exports.lcm = exports.isPrime = exports.isEven = exports.isDivisible = exports.gcd = exports.fibonacci = exports.factorial = exports.degreesToRads = exports.average = void 0;
var average_1 = require("./average");
Object.defineProperty(exports, "average", { enumerable: true, get: function () { return __importDefault(average_1).default; } });
var degreesToRads_1 = require("./degreesToRads");
Object.defineProperty(exports, "degreesToRads", { enumerable: true, get: function () { return __importDefault(degreesToRads_1).default; } });
var factorial_1 = require("./factorial");
Object.defineProperty(exports, "factorial", { enumerable: true, get: function () { return __importDefault(factorial_1).default; } });
var fibonacci_1 = require("./fibonacci");
Object.defineProperty(exports, "fibonacci", { enumerable: true, get: function () { return __importDefault(fibonacci_1).default; } });
var gcd_1 = require("./gcd");
Object.defineProperty(exports, "gcd", { enumerable: true, get: function () { return __importDefault(gcd_1).default; } });
var isDivisible_1 = require("./isDivisible");
Object.defineProperty(exports, "isDivisible", { enumerable: true, get: function () { return __importDefault(isDivisible_1).default; } });
var isEven_1 = require("./isEven");
Object.defineProperty(exports, "isEven", { enumerable: true, get: function () { return __importDefault(isEven_1).default; } });
var isPrime_1 = require("./isPrime");
Object.defineProperty(exports, "isPrime", { enumerable: true, get: function () { return __importDefault(isPrime_1).default; } });
var lcm_1 = require("./lcm");
Object.defineProperty(exports, "lcm", { enumerable: true, get: function () { return __importDefault(lcm_1).default; } });
var primes_1 = require("./primes");
Object.defineProperty(exports, "primes", { enumerable: true, get: function () { return __importDefault(primes_1).default; } });
var radsToDegrees_1 = require("./radsToDegrees");
Object.defineProperty(exports, "radsToDegrees", { enumerable: true, get: function () { return __importDefault(radsToDegrees_1).default; } });
